Apps/frontend/src/views/ControlPanelBottom.svelte

```diff
@@ -33,11 +41,11 @@
       Channel
     {/if}
   </th>
-  <th>Start/Stop</th>
-  <th>Thickness</th>
-  <th>Offset</th>
-  <th>Time Sweep</th>
-  <th>Amplitude</th>
+  <th><ControlColumnHeader label="Start/Stop" icon="toggle" tooltip={TOOLTIP_CONTROL_HEADER_ONOFF} /></th>
+  <th><ControlColumnHeader label="Thickness" icon="priority" tooltip={TOOLTIP_CONTROL_HEADER_THICKNESS} /></th>
+  <th><ControlColumnHeader label="Offset" icon="vertical" tooltip={TOOLTIP_CONTROL_HEADER_OFFSET} /></th>
+  <th><ControlColumnHeader label="Time Sweep" icon="horizontal" tooltip={TOOLTIP_CONTROL_HEADER_TIMESWEEP} /></th>
+  <th><ControlColumnHeader label="Amplitude" icon="expand" tooltip={TOOLTIP_CONTROL_HEADER_AMPLITUDE} /></th>
```

Comment:
Thought about reusing the distribute icon instead of introducing a new `expand` icon, but I actually think I like it for scaling purposes.

One thing I'd prefer to be changed, on the other hand, is the name of the icons. We could simply call the icon the same as the header text (Offset -> icon--offset)

Reply:
I did not name them as the label by purpose: the icons could be used elsewhere. But I changed it.